server/pbenchinacan/container-build-sa.sh

```diff
@@ -44,17 +44,17 @@ fi
 container=$(buildah from --format=docker ${BASE_IMAGE})
 
 # Provide a few useful defaults for labels and the host name to use, and expose
-# port 80 for the Apache2 httpd process acting as a proxy.
+# port 8080 for the Nginx proxy server.
```

Comment:
Really "Nginx"?? Or should it be "NGINX" or "nginx" to match the package name?

Reply:
Well, the previous reference was "Apache2 httpd", which was the name of the software and the name of the daemon that was run.  It seemed "Nginx nginx" was from the department of the redundancy department, so I figured "Nginx" is sufficient.

Reply:
Yeah, I get that: I was just questioning the case conventions here. Do people really use mixed-case "Nginx" commonly?

Reply:
Yes, at least in my experience of using Nginx since 2007.